src/test/java/org/mockito/internal/stubbing/defaultanswers/foopackage/ForwardsInvocationsTest.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2017 Mockito contributors
+ * This program is made available under the terms of the MIT License.
+ */
+package org.mockito.internal.stubbing.defaultanswers.foopackage;
+
+import org.junit.Test;
+import org.mockito.internal.stubbing.defaultanswers.ForwardsInvocations;
+import org.mockitoutil.TestBase;
+
+import static org.junit.Assert.assertEquals;
+
+public class ForwardsInvocationsTest extends TestBase {
```

Comment:
Do we need this test? Does this test improves coverage on top of the existing ForwardsInvocationsTest?

Reply:
The original problem was the inability to mock anonymous classes. In line 21 of this test, an anonymous class is created. I think this is therefore testing the appropriate codepath?

Reply:
It is. Please disregard my comment.

Reply:
Yeah, our comments were cross-posted. I saw your reply the second I posted my comment 😛